app/mapping/registration/TrustDetailsMapper.scala

```diff
@@ -71,9 +70,9 @@ class TrustDetailsMapper extends Mapping[TrustDetailsType] {
     } yield {
       registration.TrustDetailsType(
         typeOfTrust = typeOfTrust,
-        deedOfVariation = deedOfVariation(userAnswers),
-        interVivos = userAnswers.get(HoldoverReliefYesNoPage),
-        efrbsStartDate = userAnswers.get(EfrbsStartDatePage)
+        deedOfVariation = ???, //deedOfVariation(userAnswers),
```

Comment:
Again does this need an implimentation or is it handled in settlors and can be removed?